overalls.go

```diff
@@ -214,8 +214,8 @@ func processDIR(logger *log.Logger, wg *sync.WaitGroup, fullPath, relPath string
 	// 1 for "test", 4 for covermode, coverprofile, outputdir, relpath
 	args := make([]string, 1, 1+len(flagArgs)+4)
 	args[0] = "test"
-	args = append(args, flagArgs...)
 	args = append(args, "-covermode="+coverFlag, "-coverprofile="+pkgFilename, "-outputdir="+fullPath+"/", relPath)
+	args = append(args, flagArgs...)
```

Comment:
There is one thing that needs to be checked first there are additional args that can be passed to go test, but this change moves all those arguments to the package level; I think on line 217 the args need to be checked for `-args` and averything ebfore that arg gets appended before the `-covermode` etc and the rest after

eg.
```go
for i,str:=range flagArgs {
  if arg == "-args"{
    flagArgs = flagArgs[:i]
    break
  }
  args = append(args,arg)
}
}
```